src/main/java/com/example/animalshelter/model/Animal.java

```diff
@@ -0,0 +1,23 @@
+package com.example.animalshelter.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * Keeps basic information about an animal
+ */
+@Getter
+@Setter
+@ToString
+public class Animal {
+
+    private int id;
```

Comment:
I think this value should not be set by the user. Maybe should be static?

Reply:
Yes, agree we shouldn't set up id by us own. Probably it should be set up automatically by framework. We need to add specific annotation later to support that.

Reply:
Ok.